Fastfile.private

```diff
@@ -78,12 +79,12 @@ reflected in `Info.plist` during building."
     ).to_i
 
     bump_type = check_bump_type(
-      build_configuration: build_configuration,
+      build_configuration: build_configuration_key,
       version: current_version_version,
```

Comment:
Funny this is called `current_version_version`, maybe `current_version_number`? (Same for build).

Reply:
Yes, that name is awful, I'll change it.